home/views.py

```diff
@@ -168,18 +174,27 @@ def update_cwhw(request):
         section_id = request.POST['section']
         section = Section.objects.get(section_id=section_id)
 
+        to_be_taught = request.POST['extra-info-2']
```

Comment:
`to_be_taught` is a whole different field, you don't need to do anything to that.